python/tvm/relay/quantize/_annotate.py

```diff
@@ -183,14 +184,20 @@ def conv2d_rewrite(ref_call, new_args, ctx):
 def dense_rewrite(ref_call, new_args, ctx):
     """Rewrite function for dense. Lhs of dense will be quantized to input field, and rhs of
     dense will be quantized to weight field. Output would be in activation field."""
-    cnt = _conv_counter()
-    if cnt < current_qconfig().skip_k_conv:
+    cnt = _dense_counter()
+
+    if cnt < current_qconfig().skip_k_dense:
+        _set_conv_counter(cnt + 1)
         return None
-    if current_qconfig().skip_conv_layers is not None:
-        leave_alone_indices = [int(x) for x in current_qconfig().skip_conv_layers]
-        if cnt - 1 in leave_alone_indices:
+
+    if current_qconfig().skip_dense_layers is not None:
```

Comment:
Can we also put a todo to improve the way we handle the "skipping" it seems ad-hoc right now.

Reply:
Agreed, there should be a better way than skipping; a pattern matching approach should be more future-proof IMO, I think we can start an RFC after we merge this in for now.